tests/Unit/UtilsTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+namespace Automattic\LegacyRedirector\Tests\Unit;
+
+use Automattic\LegacyRedirector\Utils;
+use Brain\Monkey;
+use Yoast\WPTestUtils\BrainMonkey\TestCase;
+
+/**
+ * Capability Class Unit Test
+ */
+final class UtilsTest extends TestCase {
+
+	/**
+	 * Utils::mb_parse_url test method
+	 *
+	 * @covers \Automattic\LegacyRedirector\Utils::mb_parse_url
+	 */
+	public function test_mb_parse_url() {
+
+		Monkey\Functions\stubs(
+			array(
+				'wp_parse_url' => static function ( $url, $component ) {
+					return parse_url( $url, $component );
+				},
+			)
+		);
```

Comment:
This stub is also defined in MonkeyStubs - does there need to be the duplicate?

Reply:
added a fix here to get the stub from the MonkeyStub class, I see it used in the PreservableParamsTest test as well, so we might as well just share it.